### ISSUE TITLE
does model.fit has a loss variable to monitor?

### ISSUE THREAD

**Issue by Mshz2** [rvorias/ind_knn_ad#19]
Hi there,
I was wondering if inside [model.py](https://github.com/rvorias/ind_knn_ad/blob/master/indad/models.py) is there a loss or cost value where it would change during training or fitting? 
- Or are we even retrain the backbone to the new datasets? is there even learning of backbone in the pipeline where you specify number of epochs?
I am particularly talking about PADIM. Thanks a lot for your support.
Best

**Comment by rvorias** [comment]
Hi, there is no loss function, we are just storing the dataset.
